File: abc/abc309/a/main.dart
import 'dart:io';

void main() {
  final inputs = stdin.readLineSync()?.split(" ")?.map(int.parse);
  final a = inputs?.first;
  final b = inputs?.last;

  if (a?.remainder(3) == 0) {
    print('No');
  } else if (b != null && a != null && b - a == 1) {
    print('Yes');
  } else {
    print('No');
  }
}

File: abc/abc309/a/main.ts
import * as fs from "fs";

const inputs = fs.readFileSync(process.stdin.fd, "utf8").trim();

const [a, b] = inputs.split(" ").map(value => parseInt(value));

const next = Math.abs(a - b);

if (next > 1) {
    console.log("No");
}

if (next === 1) {
    if (a % 3 === 0 && b % 3 === 1) {
        console.log("No");
    } else {
        console.log("Yes");
    }
}